nat-lab/tests/test_node_state_flickering.py

```diff
@@ -3,6 +3,7 @@
 import telio
 from contextlib import AsyncExitStack
 from helpers import SetupParameters, setup_mesh_nodes
+from telio_features import TelioFeatures, Direct
```

Comment:
Do I understand correctly that the tests in this file would fail now (with direct enabled), without the changes in `uapi.rs`?

Reply:
Yeah, that's the premise. However, it seems to me that it might not be the case :shrug: and I'm still trying to confirm that.
However, the preshared keys parsing logic was flawed and needs to be fixed